### DISCUSSION TITLE
"Talkgroup" and "Name" are the same

### DISCUSSION THREAD

**Discussion by hoffadg** [chuot/rdio-scanner#303] [Q&A] [answered]
I just started playing around with RDIO-scanner today and I love it so far!  It's working wonderfully but there is one thing I have not been able to figure out.  Is it possible to pull both the "Talkgroup" and "Name" strings from SDRTrunk?  As of now, they are both coming in as the same string.

![image](https://user-images.githubusercontent.com/33765756/210189315-703e25a1-3a9f-4d6a-a670-96723c7e09b2.png)

Thanks,
Dan

**Comment by GitGoorilla** [comment, marked as answer]
Rdio scanner only ingests the aliases from SDRTrunk playlist. There isn't a "name" string that is part of the playlist in SDRTrunk. The easiest way to change the name is through the Admin panel. Systems>>Carver/Scott County Fire>>Talkgroups. For instance, your Talkgroup ID would be "CSKA FD ADM" and the label would be "Chaska Fire Dept Admin". 

You could change the SDRTrunk alias name to "Chaska Fire Dept Admin" and it would populate it as that but I find it easy to just go in and change the label field in the admin panel and preserve the TG ID.

**Reply by hoffadg** [reply]
Thanks, sounds like it isn't possible to do automatically which is what I suspected.  I ended up turning off the auto populate feature and just uploaded the .csv I got from radioreference and that seemed to populate all of the fields.

**Comment by FF4405** [comment]
Has anyone had issues with the names and label's continuously being overwritten, after updating manually or importing from RR?

**Reply by GitGoorilla** [reply]
Do you have auto-populate enabled? If I remember correctly, I ran into the same issue awhile back when trying to rename the Talkgroup labels. Every time it ingested that specific label, it would rename it. Turning off auto-populate corrected it.

**Reply by FF4405** [reply]
I have it turned off.

**Reply by FF4405** [reply]
I think I found my problem in global options:

![image](https://github.com/chuot/rdio-scanner/assets/107078769/5d776f2b-6d41-49f2-99eb-310ea0c6e0a1)

Turned this on off.

**Reply by GitGoorilla** [reply]
Yep. That would be it. There's a global option and a system specific option.
